driver/razeranansicommon.c

```diff
@@ -0,0 +1,14 @@
+#include "razeranansicommon.h"
+
+struct razer_report razer_anansi_enable_macro_flash()
+{
```

Comment:
This works for making the macro led blink as well. I tried changing argument 2, and the state of the macro led didn't change.

```c
struct razer_report report = get_razer_report(0x03, 0x04, 0x04);
report.arguments[0] = variable_storage;
report.arguments[1] = led_id;
report.arguments[2] = 0x05;
report.arguments[3] = 0x01;

```